conf/up4.bess

```diff
@@ -99,7 +100,7 @@ else:
 
 
 # Initialize workers
-cores = get_process_affinity()
```

Comment:
Given that with this change the `get_process_affinity` function is not used anymore, should it be removed from `utils.py`?

Reply:
Excellent point. I was not sure about if it could have been used elsewhere without me knowing it.

Reply:
I do not see it used anywhere else